frontend/src/index.js

```diff
@@ -37,9 +38,18 @@ const authMiddleware = new ApolloLink((operation, forward) => {
   return forward(operation);
 });
 
+const logoutLink = onError(({ networkError }) => {
+  console.log(networkError);
+  if (networkError && process.env.REACT_APP_OKTA_URL) {
+    console.error("network error", networkError);
+    console.log("redirecting to", process.env.REACT_APP_OKTA_URL);
+    window.location.replace(process.env.REACT_APP_OKTA_URL);
+  }
+});
+
 const client = new ApolloClient({
   cache: new InMemoryCache(),
-  link: concat(authMiddleware, httpLink),
+  link: logoutLink.concat(concat(authMiddleware, httpLink)),
```

Comment:
&lt;boggle&gt;
I have no idea what's happening here. Time to learn a new stack component! (Except actually right now is definitely _not_ that time.)